docs/src/conventions.md

```diff
@@ -0,0 +1,116 @@
+# Coding Conventions
+
+## Formatting
+
+We automate most of our formatting preferences. Our CI will run format checkers for PRs and pushes.
+These checks are required for merging any PR.
+
+For rust, we use [rustfmt](https://github.com/rust-lang/rustfmt)
+which is configured via [rustfmt.toml](https://github.com/model-checking/kani/blob/main/rustfmt.toml) file.
+We are also in the process of enabling clippy.
+Because of that, we still have a couple of lints disabled (see [.cargo/config](https://github.com/model-checking/kani/blob/main/.cargo/config.toml) for the updated list).
+
+We also have a bit of C code and python in our repository.
+For C we use clang-format and for python scripts we use autopep8.
```

Comment:
I know this is just to document the status quo, but out of curiosity: is there a reason not use [black](https://black.readthedocs.io/en/stable/) for Python? I found it quite pleasant because it completely eliminates any discussion about coding style.

Reply:
TBH, we just went with the standard approach. When we enabled the python format checker we were already planning to move out of python, so we didn't spend much time researching alternatives.